stage0/src/dice.rs

```diff
@@ -0,0 +1,81 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+use crate::image_header::Image;
+use core::str::FromStr;
+use dice_crate::{
+    AliasCert, AliasData, AliasOkm, Cdi, CdiL1, DeviceIdCert, DeviceIdOkm,
+    Handoff, SeedBuf, SerialNumber,
+};
+use lpc55_pac::Peripherals;
+use salty::signature::Keypair;
+use sha3::{Digest, Sha3_256};
+use unwrap_lite::UnwrapLite;
+
+fn get_deviceid_keypair(cdi: &Cdi) -> Keypair {
+    let devid_okm = DeviceIdOkm::from_cdi(cdi);
+
+    Keypair::from(devid_okm.as_bytes())
+}
+
+fn get_serial_number() -> SerialNumber {
+    // get serial number from somewhere
```

Comment:
Mark this with a TODO (and file an issue) so we don't forget.

Reply:
https://github.com/oxidecomputer/hubris/issues/734